### DISCUSSION TITLE
Year is no longer retrieved when searching for a movie.

### DISCUSSION THREAD

**Discussion by arivacoba** [cinemagoer/cinemagoer#455] [Q&A] [answered]
Previously, when searching for a movie, each of the results would contain the year, when available. It is not longer the case, what may I do to revert to the previous behavior?

In example, this code used to work:

> from imdb import Cinemagoer
> mp = imdb.Cinemagoer()
> def BuscaTitYear(titulo, anio):
>     titulos = mp.search_movie(titulo)
>     while titulos:
>         titul=titulos.pop(0)
>         if 'year' in titul and str(titul['year']) == anio:
>             if titul['title'].upper() == titulo.upper() or (titul.has_key('akas') and titulo.upper() in [x.upper() for x in titul['akas']]):
>                 consola.info ("Coincidencia encontrada %s", repr(titul))
>                 return (titul)
>     consola.critical ("Except: No se puede determinar la pelicula en IMDb - \n%s",dest)

That is currently failing, as there is no 'year' key in the titles. For example:

> titulos=mp.search_movie('star wars')
>    ... titulos contains a list of movie objects ...
> titulos[0]
>    ... <Movie id:0076759[http] title:_Star Wars: Episode IV - A New Hope (None)_>
> titulos[0].keys()
>    ... ['title', 'kind', 'cover url', 'canonical title', 'long imdb title', 'long imdb canonical title', 'smart canonical title', 'smart long imdb canonical title', 'full-size cover url']
> 

I appreciate your comments. 
Aaron

Python 3.10.6, Cinemagoer (2022.12.27)

**Comment by alberanid** [comment, marked as answer]
it should be fixed; please update to the latest release.

Reopen the issue if you still have problems.

**Reply by arivacoba** [reply]
Thank you very much. 

It is indeed corrected by updating to 2023.05.01
